### ISSUE TITLE
Add <WBN324 Talk Radio> to the list of merchants

### ISSUE THREAD

**Issue by 1Skye1** [monero-project/monero-site#1616]
#### Enter the URL where a Monero mention can be found

https://reabow.wixsite.com/sundaylonglive

#### How do you accept Monero? *(choose only one option)*

- [ ] Monero integrations *(github.com/monero-integrations)*
- [ ] Globee
- [ *] Native wallet
- [ ] Other payment gateaway (Coinpayments, Morphtoken, etc. *Please specify*)

#### How would you categorize your business? *(choose only one option)*

- [ ] Exchange
- [ ] Block Explorer
- [ ] Payment Gateway
- [ ] Web Hosting
- [ ] Tool
- [ *] Service
- [ ] Goods
- [ ] Entertainment

**Comment by erciccione** [comment]
> WBN324 Talk Radio is Monero friendly and offering cheap advertising paid with Mornero

This is not max 50 characters.

**Comment by 1Skye1** [comment]
> > WBN324 Talk Radio is Monero friendly and offering cheap advertising paid with Mornero
> 
> This is not max 50 characters.

 Appoligies, Ive just edited it. .

**Comment by erciccione** [comment]
OK.

**Comment by 1Skye1** [comment]
HI,  I'm just checking to see if we can get listed?   Thanks. .

**Comment by erciccione** [comment]
Yes. We will add this to the website with the next round of merges.

**Comment by 1Skye1** [comment]
OK, Thanks,  FYI, We are putting togther a head to head show with Pirate chain vs Monero.  I'm in Monero camp, and another host is bringing on someone from pirate chain. The audience largey knows nothing about crypto, nevermind privavy coins. So we think its a win all round. .   Do you know anyone who would be good on air, and can represent Monero?  I can be contacted at wbn324@protonmail.com 

Thanks. . 

Rea. .

**Comment by erciccione** [comment]
#1651
